src/apps/chifra/internal/globals/output.go

```diff
@@ -131,7 +147,7 @@ func (opts *GlobalOptions) RenderFooter() error {
 			b, err := json.MarshalIndent(meta, "  ", "  ")
 			opts.Writer.Write(b)
 		}
-		opts.Writer.Write([]byte("\n}"))
+		opts.Writer.Write([]byte("\n}\n"))
```

Comment:
Let's put a TODO list item here. I think that we can "correct the test cases" as opposed to do this only to make the test cases work. I'm not sure if the right solution is "no" extra line feed or "yes" extra line feed, but let's correct the gold test cases as opposed to generating less "canonical" data. But don't let this delay your work -- thus the TODO maker.

Reply:
Sure. One reason to have the newline is printing JSON on command line. Without new line, `}` appears just before the shell prompt (on the same line). It's little frustrating :P